src/java.base/share/classes/java/util/Properties.java

```diff
@@ -1347,7 +1348,7 @@ public synchronized String toString() {
     }
 
     @Override
-    public Set<Object> keySet() {
+    public Set<@KeyFor("this") Object> keySet() {
```

Comment:
@smillst 
I didn't find these missing annotations in any of Properties.java of JDK8 i.e. I looked at following files and didn't find the annotations `@KeyFor` on any element.

 - nullness/java/util/Properties.java
- lock/java/util/Properties.java
- index/java/util/Properties.java.

Can you please direct me where to find these annotations in JDK8?
Thanks.

Reply:
No, you're right, those weren't missing.  (I forgot that I had added them to checker/jdk/nullness/ in my branch.)  I had to add them because in jdk 8 Properties did not override entrySet and keySet, but it does in jdk 11.  This is going to be a general problem for methods added in 9+.